cluster/src/main/java/org/apache/iotdb/cluster/ClusterMain.java

```diff
@@ -157,6 +157,7 @@ private static void startServerCheck() throws StartupException {
               config.getSeedNodeUrls().size(), quorum);
       throw new StartupException(metaServer.getMember().getName(), message);
     }
+    logger.info("seed urls in start Server Check 3:" + config.getSeedNodeUrls());
```

Comment:
What this log is used for?

Reply:
It is a very very strange error... I will remove it after double checking.